### PR TITLE
Add more use case to check validation of invalid url

### PR DESCRIPTION
**Pull request by mgold1234**
related to THEEDGE-1210

Signed-off-by: mgold1234 <mgold@redhat.com>

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [x] Tests update
- [ ] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

File: pkg/services/thirdpartyrepo_test.go [hunk 1]
```diff
@@ -71,13 +72,19 @@ var _ = Describe("ThirdPartyRepos basic functions", func() {
...
 			Expect(err).To(HaveOccurred())
 			Expect(err.Error()).To(Equal("invalid URL"))
-		})
+		},
+			Entry("validate invalid URL with : missing", "http//google.com"),
+			Entry("validate invalid URL with https:// missing", "google.com"),
+			Entry("validate invalid URL without https:// and . missing", "foo/bar"),
+			Entry("validate invalid URL with number", "5432/bar"),
+			Entry("validate invalid URL with symbols", "http//:&%$#@bar"),
+		)
```
Comment (ldjebran) [inline pkg/services/thirdpartyrepo_test.go:87]: "http://invalid-internet-host"  will pass validation

Comment (mgold1234) [reply pkg/services/thirdpartyrepo_test.go:87]: On the UI  we check if the url starts with https:// and is a valid url and don't check the url response.
maybe do you have an idea to solve the response of url?

Comment (ldjebran) [reply pkg/services/thirdpartyrepo_test.go:87]: I am not here talking about response problem at all.

only the host name is invalid , it should be of the form of :  foo.bar

Comment (ldjebran) [reply pkg/services/thirdpartyrepo_test.go:87]: so http://invalid-internet-host/llll   should not pass validation
bout "http://valid-internet-host.com/llll  should pass validation

Comment (ldjebran) [reply pkg/services/thirdpartyrepo_test.go:87]: you can make some testing here  https://www.regextester.com/94502

**Pull request merged**